services/rfq/relayer/quoter/quoter_test.go

```diff
@@ -202,6 +202,248 @@ func (s *QuoterSuite) TestIsProfitable() {
 	s.False(s.manager.IsProfitable(s.GetTestContext(), quote))
 }
 
+func (s *QuoterSuite) TestGetOriginAmountActiveQuotes() {
```

Comment:
May be easier to add some test cases with the DepositAmount to the existing TestGetOriginAmount() test, but up to you

Reply:
i originally did that, but the function was huge & mostly cloned from the existing ones. felt cleaner to separate